avm/utilities/tools/Invoke-WorkflowsForBranch.ps1

```diff
@@ -172,6 +172,11 @@ Optional. The GitHub repository to run the workfows in.
 Invoke-WorkflowsForBranch -PersonalAccessToken '<Placeholder>' -TargetBranch 'feature/branch' -PipelineFilter 'avm.res.*' -WorkflowInputs @{ staticValidation = 'true'; deploymentValidation = 'true'; removeDeployment = 'true' }
 
 Run all GitHub workflows that start with'avm.res.*' using branch 'feature/branch'. Also returns all GitHub status badges.
+
+.EXAMPLE
+Invoke-WorkflowsForBranch -PersonalAccessToken '<Placeholder>' -TargetBranch 'feature/branch' -PipelineFilter 'avm.res.*' -WorkflowInputs @{ staticValidation = 'true'; deploymentValidation = 'true'; removeDeployment = 'true' } -WhatIf
+
+Only simulate the triggering of all GitHub workflows that start with'avm.res.*' using branch 'feature/branch'. Hence ONLY returns all GitHub status badges.
```

Comment:
So it returns the status badge of the previous run?

Reply:
Yes-ish. Not the 'previous' per se but just the ones for what it would trigger. So if you provide all properties as you would for triggering for a branch 'Rick' - and provide the `WhatIf` you get only the status badges for pipeline runs for the branch 'Rick'.
However, nothing stops you from providing any other branch or pipeline filter to generate just the targeted pipeline badges. 
I hope that makes sense :D